src/utils/CAPTCHAHandler.ts

```
@@ -0,0 +1,450 @@
+import { Page } from '@playwright/test'
+import { CAPTCHAConfig, CAPTCHAHandlingResult } from '../types'
+import { CAPTCHADetection, CAPTCHADetector } from './CAPTCHADetector'
+import { CAPTCHASolver } from './CAPTCHASolver'
+import { PathManager } from './PathManager'
+import { sleep } from './sleep'
+
+export class CAPTCHAHandler {
+    private solver: CAPTCHASolver
+    private config: CAPTCHAConfig
+    private static isInCleanup: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)